core/sql/regress/compGeneral/EXPECTED045

```diff
@@ -0,0 +1,1606 @@
++>from (
++>      select channel, i_brand_id, i_class_id, i_category_id, sum_sales, number_sales from results
++>      union
++>      select channel, i_brand_id, i_class_id,  null as i_category_id, sum(sum_sales), sum(number_sales) from results
++>      group by channel, i_brand_id, i_class_id
++>      union
++>      select channel, i_brand_id, null as i_class_id, null as i_category_id, sum(sum_sales), sum(number_sales) from results
++>      group by channel, i_brand_id
++>      union
++>      select channel, null as i_brand_id, null as i_class_id, null as i_category_id, sum(sum_sales), sum(number_sales) from results
++>      group by channel
++>      union
++>      select null as channel, null as i_brand_id, null as i_class_id, null as i_category_id, sum(sum_sales), sum(number_sales) from results) z
++>order by channel, i_brand_id, i_class_id, i_category_id
++> limit 100 ;
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Attempting to unnest Subquery)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[2997]  (Subquery was not unnested. Reason: No Correlation found)
+
+*** WARNING[1214] Error Error in creating temp table or temp table insert encountered when executing HiveQL statement ../optimizer/NormRelExpr.cpp.
```

Comment:
Wow, you win three bonus points :-)
The first for reviewing the expected file, the second for finding a bug with ComDiagsArea::negateAllErrors(). The error message and the parameters are mixed up here, the text belongs to another error entry.
Third bonus point for pointing out another issue with the Hive table - Hive doesn't support DECIMAL data types with precision greater than 38, and in this case we want a DECIMAL(57,19).

So, thanks very much for pointing out 3 issues-in-1!